pubspec.yaml

```diff
@@ -51,6 +51,9 @@ dependencies:
   video_player: ^2.2.7
   visibility_detector: ^0.2.2
 
+dependency_overrides:
```

Comment:
Why is this necessary? The uses standard `ColorScheme` class.

Reply:
It's been added to prevent platform from having errors such as `io.Platform.packageRoot` error. Without it, debugging won't run.

Reply:
Please create another issue to resolve the `io.Platform.packageRoot` error